e2e-tests/insurance/cypress/e2e/journeys/policy/loss-payee-details/validation/loss-payee-details-validation.spec.js

```diff
@@ -87,6 +88,14 @@ context('Insurance - Policy - Loss Payee Details - Validation', () => {
 
       cy.submitAndAssertFieldErrors(field, value, errorIndex, numberOfExpectedErrors, ERROR.ABOVE_MAXIMUM);
     });
+
+    it(`should not render validation errors when ${FIELD_ID} contains numbers and special characters`, () => {
+      cy.keyboardInput(fieldSelector(FIELD_ID).input(), 'Name 123!');
```

Comment:
Can we use `mockNameWithSpecialCharacters` for this value

Reply:
I think we should test that when going back to the page, special characters are rendered as expected, we have similar tests e.g

```js
submit-policy-textarea-fields-with-special-characters.spec.js
submit-name-fields-with-special-characters/*.spec.js
```